sdk/ml/azure-ai-ml/swagger/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview/machineLearningServices.json

```diff
@@ -2259,6 +2259,53 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/managedNetwork/provisions": {
```

Comment:
this path doesn't make senses for me. Please work with Kevin to correct this one.

Reply:
Improved path to be {workspaceName}/provisionManagedNetwork